.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  extends: [require.resolve('@backstage/cli/config/eslint')],
+  rules: {
+    // I'd be for this rule, but nest.js kind-of enforces using classes more than using functions,
+    // so we'll turn this off for now.
+    'notice/notice': 'off',
```

Comment:
can we remove this rule? given we're not using nest.js directly as far as I can see (I'd imagine this is for backstage backend code)

Reply:
I think this comment (about nest.js) is irrelevant for the rule and is unnecessary. Marek probably copied it from pull requests plugin I think. This rule was turned off by me in pull requests plugin on David's request.

Reply:
So should I just delete .eslintrc.js file?

Reply:
Deleted

Reply:
I think just the comment actually? we want the rule set. If there are no errors triggered by 'notice/notice' we can remove the "rules" overrides altogether.